---
title: CredentialsNotFoundError
description: API reference for qiskit.providers.ibmq.credentials.CredentialsNotFoundError
in_page_toc_min_heading_level: 1
python_api_type: exception
python_api_name: qiskit.providers.ibmq.credentials.CredentialsNotFoundError
---

# CredentialsNotFoundError

<span id="qiskit.providers.ibmq.credentials.CredentialsNotFoundError" />

`CredentialsNotFoundError(*message)`

Errors raised when the credentials are not found.

Set the error message.

### with\_traceback

<span id="qiskit.providers.ibmq.credentials.CredentialsNotFoundError.with_traceback" />

`with_traceback()`

Exception.with\_traceback(tb) â€“ set self.\_\_traceback\_\_ to tb and return self.

